pkg/state/state_shared_dp.go

```diff
@@ -68,7 +68,6 @@ func (s *stateSharedDp) Sync(customResource interface{}, infoCatalog InfoCatalog
 	if cr.Spec.RdmaSharedDevicePlugin == nil {
 		// Either this state was not required to run or an update occurred and we need to remove
 		// the resources that where created.
-		// TODO: Support the latter case
```

Comment:
why did you remove the TODO?

Reply:
It should be removed now. It was supposed to support DevicePlugin config update but after the discussion with @adrianchiris we agreed that it should be done in other way

Reply:
so  you still have the TODO on the state_sriov_dp.go in Line 71

Reply:
I'm not sure that it will be done in these files. My proposal is to keep it as is and track in #102

Reply:
I understand so just remove the comment from state_sriov_dp.go as well

Reply:
ok, will do it